pixel_pointcloud:
  
  image_width: 1280
  image_height: 720
  camera_instrinsics: 
    [ 410.2170434107159, 0.       , 624.1044122845125,
      0.       , 410.1185305193511, 430.5091396616502,
      0.       , 0.     , 1.      ]
  distortion_coefficients: [-0.02049934577987179, -0.04794362026186617, 0.05970460237983495,-0.02485701563224011]
  # camera_instrinsics: 
  #   [ 906.404, 0.       , 632.699,
  #     0.       , 905.613, 356.749,
  #     0.       , 0.     , 1.      ]
  # distortion_coefficients: [0, 0, 0, 0, 0]

  # image_width: 640
  # image_height: 480  
  # camera_instrinsics:
  #         [ 280.66603,    0.     ,  317.56932,
  #             0.     ,  279.09595,  312.51635,
  #             0.     ,    0.     ,    1.     ]
  # distortion_coefficients: [-0.196668, 0.022176, 0.003651, -0.001057, 0.000000]
  
  camera_lidar_extrinsic: 
    [ 9.981942754678563290e-03, -9.998453176383657048e-01, -1.448107784241975393e-02, 3.796365770536530571e-02,
      -4.689828948594672031e-01, 8.109147755389622958e-03, -8.831700210332750212e-01, 4.608911448604903482e-02,
      8.831508394085783031e-01, 1.560713039982564165e-02, -4.688294064290405294e-01, -2.478778661740722172e-03,
      0.000000000000000000e+00, 0.000000000000000000e+00, 0.000000000000000000e+00, 1.000000000000000000e+00 ]
  
  
  # image_raw_input: "/camera/color/image_raw"
  image_raw_input: "/usb_cam0/image_raw"
  # image_rect_output: "/image_rect"
  cloud_raw_input: "/livox/lidar"
  